index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>JS Challenges</title>
+  </head>
+
+  <body>
+    <main id="main-doc">
+      <div class="box">
+        <nav id="navbar">
+          <header class="h1">JS Challenges</header>
+          <a href="#Apple_and_Orange" class="nav-link">Apple and Orange</a>
+          <a href="#Breaking_best_and_worst_records" class="nav-link"
+            >Breaking best and worst records</a
+          >
+          <a href="#Cats_and_a_mouse" class="nav-link">Cats and a mouse</a>
+          <a href="#The_hurdle_race" class="nav-link">The hurdle race</a>
+          <a href="#Simple_Array_Sum" class="nav-link">Simple Array Sum</a>
+        </nav>
+      </div>
+
+      <div class="container">
+        <section class="main-section" id="Apple_and_Orange">
+          <header><h2>Apple and Orange</h2></header>
+          <p>
+            Sam's house has an apple tree and an orange tree that yield an
+            abundance of fruit. Using the information given below, determine the
+            number of apples and oranges that land on Sam's house.
+          </p>
+          <div class="code">
+            <p>
+              <code>
+                function countApplesAndOranges(s, t, a, b, apples, oranges) {
+                let positionApples = []; let positionOranges = [];
+                apples.forEach( x => {if((x+a) >= s && (x+a) &lt;= t) return
+                positionApples.push(x+a)}); oranges.forEach( x => {if((x+b) >= s
+                && (x+b) &lt;= t) return positionOranges.push(x+b)});
+                console.log(positionApples.length);
+                console.log(positionOranges.length); }
+              </code>
+            </p>
+          </div>
```

Comment:
It would be better if you remove the p tag because it doesn't have any influence on your HTML structure. The code tag will do that work. :v: